src/blockstore.js

```diff
@@ -85,26 +85,21 @@ function createBaseStore (store) {
     /**
      * Like put, but for more.
      *
-     * @param {Array<Block>} blocks
+     * @param {AsyncIterable<Block>|Iterable<Block>} blocks
      * @returns {Promise<void>}
      */
     async putMany (blocks) {
```

Comment:
The Readme needs to be updated to reflect the change, otherwise this looks good.

Reply:
Done!